compiler+runtime/include/cpp/jank/runtime/detail/native_array_map.hpp

```diff
@@ -41,7 +41,7 @@ namespace jank::runtime::detail
 
     void erase(object_ref const key);
 
-    object_ref find(object_ref const key) const;
+    object_ref *find(object_ref const key) const;
```

Comment:
We don't want to introduce raw pointers into any new APIs. They're considered very poor practice in C++, since they have no notion of ownership. If the goal is to distinguish "not found" from "nil", we should either use a `jtl::option` or at least a `jtl::ptr`.

Reply:
I switched the raw pointer to `jtl::option` 👍